package.json

```diff
@@ -146,18 +146,18 @@
     "postcss-loader": "^3.0.0",
     "postcss-preset-env": "^3.0.2",
     "precss": "^2.0.0",
-    "prettier-eslint": "^13.0.0",
-    "prettier-eslint-cli": "^5.0.0",
+    "prettier-eslint": "^16.3.0",
+    "prettier-eslint-cli": "^8.0.1",
     "raf": "^3.4.0",
     "raw-loader": "^0.5.1",
     "react-hot-loader": "^4.13.0",
     "react-refresh": "^0.11.0",
     "react-test-renderer": "^17.0.2",
     "redux-mock-store": "^1.2.3",
     "style-loader": "^0.23.1",
-    "ts-jest": "^25.0.0",
+    "ts-jest": "^26.5.6",
     "ts-loader": "^8.0.2",
-    "typescript": "^3.9.5",
+    "typescript": "^4.6.3",
```

Comment:
introducing the latest driver version also introduced some type errors at compile time

here, I've set the version of typescript to match that of the neo4j js driver